packages/bierzo-wallet/package.json

```diff
@@ -8,25 +8,63 @@
     "@fortawesome/react-fontawesome": "^0.1.4",
     "@material-ui/core": "^4.0.0-alpha.6",
     "@material-ui/styles": "^4.0.0-alpha.6",
-    "@types/jest": "24.0.11",
     "@types/node": "11.12.2",
     "@types/react": "16.8.10",
     "@types/react-dom": "16.8.3",
     "medulas-react-components": "^0.0.0",
     "normalize.css": "^8.0.1",
-    "react-scripts": "2.1.8"
+    "react-scripts": "3.0.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-react": "^7.0.0",
+    "@types/enzyme": "^3.9.1",
+    "@types/enzyme-adapter-react-16": "^1.0.5",
+    "@types/jest": "^24.0.11",
+    "babel-jest": "^24.7.1",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.12.1",
+    "jest": "^24.7.1",
+    "jest-environment-enzyme": "^7.0.2",
+    "jest-enzyme": "^7.0.2",
+    "react-testing-library": "^6.1.2",
+    "ts-jest": "^24.0.2"
   },
   "scripts": {
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
-    "start": "react-scripts start"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "start": "react-scripts start",
+    "test": "jest"
   },
   "browserslist": [
     ">0.2%",
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
```

Comment:
And this?

Reply:
Some solutions that I started to try in order to integrate the testing technologies used some kind of configuration for Babel and Jest. That config can be set in a separate file or in the package.json, like I did after this closing bracket.

Reply:
huh, using create-react-app should abstract you to all of this. I believe this is something related to enzyme.